src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -144,7 +144,7 @@ public static IEnumerable<DType> GetPrimitiveTypes()
         /// Eventually, all display names should come from this centralized source.
         /// We should not be using individual DataSource/OptionSet/View references.
         /// </summary>
-        internal DisplayNameProvider DisplayNameProvider { get; private set; }
+        internal DisplayNameProvider DisplayNameProvider { get; set; }
```

Comment:
>set [](http://example.com/codeflow?start=64&length=3)

making this mutable isn't ideal...

Reply:
Thanks for catching this. It was a previous test. I'll run another code clean-up before merging.